dirs/dirs.go

```diff
@@ -283,6 +283,11 @@ func FeaturesDirUnder(rootdir string) string {
 	return filepath.Join(rootdir, snappyDir, "features")
 }
 
+// SnapSystemParamsUnder returns the path to the system-params file under rootdir
```

Comment:
missing . at the end

Reply:
thank you for catching all my missing dots these days!